src/pages/meal-plan/meal-plan-create-page.tsx

```
@@ -151,3 +158,205 @@ const MealPlanCreatePage = () => {
+          onClick={handleClick}
+          handleModal={() => setIsOpen(false)}
+        />
+      )}
+      <p className="pl-4 pt-6 font-semibold text-[24px] text-gray-800 pb-4 whitespace-pre-line">
+        우리가족을 위한 식단표가 {"\n"}생성되었어요.
+      </p>
+      <div className="flex justify-end pr-[14px]">
+        <button
+          className="flex gap-1 p-[10px] bg-gray-200 rounded-lg font-medium text-[16px] cursor-pointer"
+          onClick={onClick}
+        >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)